src/shared/json.h

```diff
@@ -227,6 +228,7 @@ enum {
         _JSON_BUILD_LITERAL,
         _JSON_BUILD_STRV,
         _JSON_BUILD_BASE64,
+        _JSON_BUILD_HEX,
```

Comment:
would it be worth adding unit tests for this new variant in src/test/test-json.c ?

Reply:
Yes, definitely, but I am also lazy ;-)